docs/data/sensitive-data/sd-desktop-audited.md

```diff
@@ -20,11 +20,11 @@ Contents:
 
 * Only the members of the same CSC project can access the same virtual desktop.
 
-* After login to SD Desktop, the user can start a pre-built computing environment (Linux OS), on-demand; available options offer the capability of doing simple statistical analysis to machine learning.
+* After login to SD Desktop, the user can start a pre-built computing environment (Linux Ubuntu OS), on-demand; available options offer the capability of doing simple statistical analysis to machine learning.
 
 * To comply with the regulation, virtual desktops for secondary use are completely isolated from the internet and other services: you can only access the data you have requested from the data controller;
 
-* SD Desktop can be used to work with any type of data: text files, images, audio files, video, and genetic data. However, the virtual desktop includes [a limited set of pre-installed software](../../data/sensitive-data/sd-desktop-secondary-access.md#pre-installed-software) (open source). Only the permit authority can approve the use of additional software/files/scripts and import them into your virtual Desktop.
+* SD Desktop can be used to work with any type of data: text files, images, audio files, video, and genetic data. However, the virtual desktop includes [a limited set of pre-installed software](../../data/sensitive-data/sd-desktop-secondary-working.md#default-software-available-in-sd-desktop) (open source). Only the permit authority can approve the use of additional software/files/scripts and import them into your virtual Desktop.
```

Comment:
Maybe we should change this: "Only the permit authority can approve the use of additional software/files/scripts and import them into your virtual Desktop." to something like: Installing additional software to the virtual desktop is limited. Always contact servicedesk@csc.fi about software needs before starting to work with the data.